[
  {
    "project": "beartype/beartype",
    "commit": "d488ef4a9a216c9cda81a4445c33fca143242c72",
    "filename": "beartype/_decor/_code/_pep/_error/peperror.py",
    "filename_after_commit": "/home/chowyi/TypeAnnotation_Study/GitHub/beartype-beartype/beartype/_decor/_error/errormain.py",
    "file_hunks_size": 6,
    "min_patch_found": false,
    "single_hunk": false,
    "fit_TFix": false,
    "delete_only_patch": false,
    "has_suppression_all_hunks": false,
    "full_warning_msg": "beartype/_decor/_code/_pep/_error/peperror.py:294:8 Incompatible parameter type [6]: Expected `int` for 6th parameter `random_int` to call `CauseSleuth.__init__` but got `Optional[int]`.",
    "message": " Expected `int` for 6th parameter `random_int` to call `CauseSleuth.__init__` but got `Optional[int]`.",
    "rule_id": "Incompatible parameter type [6]",
    "warning_line_no": 294,
    "warning_line": "        random_int=random_int,"
  }
]